tinygrad/device.py

```diff
@@ -103,6 +103,8 @@ def _internal_buffer_copy(dest, src):
     if fb:
       dest.allocator.transfer(dest._buf, fb, dest.size*dest.dtype.itemsize)
       return
+  if getenv("LOAD_BUFFER") and hasattr(dest.allocator, 'load_buffer') and src.device.startswith("DISK"):
```

Comment:
Does this have to be gated with an env var? Is there ever a downside?

Reply:
it doesn't load tensor synchronously so you have to be aware of syncing device before usage

Reply:
according to Metal's feature set table, MTLIO is supported everywhere. so i'm not sure if a gate is required here.

<img width="1278" alt="image" src="https://github.com/tinygrad/tinygrad/assets/8665427/544e042c-1ed4-4495-9eff-724f409eb4bd">

Reply:
removed that. it was for testing purposes